### ISSUE TITLE
react story template violates jsx-a11y/accessible-emoji eslint rule

### ISSUE THREAD

**Issue by shobhitchittora** [storybooks/storybook#2736]
Recently I bootstrapped my new project using [react-boilerplate](https://github.com/react-boilerplate/react-boilerplate) and added storybook to it using `getstorybook` command. 
This worked perfectly ( Thanks! ).

React-boilerplate has precommit hooks which disallows to push code with eslint errors. It'd be great if we can play well with eslint rules and fix our boilerplate stories.

Will create a PR if we all concur.

**Comment by shobhitchittora** [comment]
Putting the error here for everyone's reference - 
```js
 error  Emojis should be wrapped in <span>, have role="img", and have an accessible description with aria-label or aria-labelledby  jsx-a11y/accessible-emoji

```

**Comment by Hypnosphi** [comment]
Fixed in `v3.4.0-alpha.9` with #3011
